packages/charts/src/components/distributed-chart-tooltip/hooks/use-distributed-tooltip.ts

```diff
@@ -105,7 +106,7 @@ const useDistributedTooltip = ({
   return {
     label: getTooltipLabel(data[recordIndex], labelSelector, data.length > 1),
     percentValue,
-    totalValue,
+    totalValue: formatValue(totalValue, tooltipSettings.formatValue) as string,
```

Comment:
Is total value should always be formatted ? Have You considered here a stack percent mode use case ?

Reply:
If we don’t provide formatValue the formatter will return unchanged value anyway. But sure I can change this to check first if current mode is percentage before applying formatter, it will be more transparent.

Reply:
Yes, please - better to have a protection for this use particular case and make sure that % values are not formatted.